### PR TITLE
porting updated ai from assignment4

### PR DESCRIPTION
**Pull request by cyan33**

### CODE REVIEW

File: src/components/AI/Vertex.js [hunk 1]
```diff
@@ -100,36 +100,36 @@ class Vertex {
   }
     
   /**
-    * Gets this vertex's location
-    * @return this vertex's location (PVector)
-    */
+     * Gets this vertex's location
+     * @return this vertex's location (PVector)
+     */
   getLoc() {
```
Comment (jegood) [inline src/components/AI/Vertex.js]: Can you fix up the javadoc spacing here? They should match up with the first '*' column. There's a couple other places in the file, too.

**Review by jegood — CHANGES_REQUESTED** [review]
A few minor things. Mostly in one file.

**Comment by cyan33** [comment]
@jegood fixed

**Review by jegood — APPROVED** [review]
LGTM

**Pull request merged**